oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```
@@ -365,6 +365,9 @@ tcp:
 AzureHttpTrigger:
   id: 111
   languages: Java,C#,Node.js,Python
+Neo4j:
+  id: 112
+  languages: Java,Golang,C#,Python,JavaScript
```

```suggestion
  languages: Java
```

We don't have plugin of neo4j in other languages.